DataFormats/SiPixelClusterSoA/src/classes_def.xml

```diff
@@ -0,0 +1,19 @@
+<lcgdict>
+  <class name="SiPixelClustersLayoutSoA"/>
+  <class name="SiPixelClustersLayoutSoA::View"/>
+  <class name="PortableHostCollection<SiPixelClustersLayoutSoA>"/>
+  <read
+    sourceClass="PortableHostCollection<SiPixelClustersLayoutSoA>"
+    targetClass="PortableHostCollection<SiPixelClustersLayoutSoA>"
+    version="[1-]"
+    source="SiPixelClustersLayoutSoA layout_;"
+    target="buffer_"
+    embed="false">
+  <![CDATA[
+    PortableHostCollection<SiPixelClustersLayoutSoA>::ROOTReadStreamer(newObj, onfile.layout_);
+  ]]>
+  </read>
+
+  <class name="SiPixelClustersHost"/>
```

Comment:
I believe this needs class version

Reply:
> I honestly don't know, but I assume if @makortel suggested adding one we should do so.

Thank you @fwyzard 
There are quite several such suggestions from @makortel also for other classes among the still unanswered comments in this PR: perhaps you should address all them, so.